python/pyarrow/array.pxi

```diff
@@ -2467,7 +2467,7 @@ cdef class MapArray(ListArray):
     """
 
     @staticmethod
-    def from_arrays(offsets, keys, items, MemoryPool pool=None):
+    def from_arrays(offsets, keys, items, DataType type=None, MemoryPool pool=None):
```

Comment:
is there a reason to use `type` as a variable name? `type` is [a built-in function in Python](https://docs.python.org/3/library/functions.html#type) and unless necessary could we call it `datatype` or `data_type`?

Reply:
Agree but at the same time, we seem to be using `type` elsewhere also. For example:
https://github.com/apache/arrow/blob/72ed58449ea71aab1343d9adce19f177f20705cf/python/pyarrow/array.pxi#L2139C15-L2145

Reply:
oh! I see, I did a search and didn't found other occurrences. Thanks @AlenkaF for pointing that out